### PR TITLE
Accessibility fixes ported from 2.22

### PR DESCRIPTION
**Pull request by asurdej-comcast**

### CODE REVIEW

**Comment by lauromoura** [comment]
LGTM overall.

Regarding the ATK bits, ATK support was removed from upstream recently (replaced with AT-SPI).

File: Source/WebCore/accessibility/AccessibilityNodeObject.cpp [hunk 1]
```diff
@@ -1451,6 +1455,18 @@ void AccessibilityNodeObject::visibleText(Vector<AccessibilityText>& textOrder)
         if (!text.isEmpty())
             textOrder.append(AccessibilityText(text, AccessibilityTextSource::Children));
     }
+
+    if(textOrder.size() == 0 && this->isTableRow()) {
```
Comment (lauromoura) [inline Source/WebCore/accessibility/AccessibilityNodeObject.cpp:1459]: Just a question before merging: If I understand correctly, this covers some downstream-specific needs not intended for upstreaming into current webkit's master, right? Or would be something that could be added to the spec? Couldn't find the WAI-ARIA spec regarding this, though.

Comment (asurdej-comcast) [reply Source/WebCore/accessibility/AccessibilityNodeObject.cpp:1459]: Initially this was fixing some specific app use-case but I think this is a pretty generic change.
There is a test page prepared for this: http://haltersweb.github.io/Accessibility/CSUN2019/ (at the top there is a expected text that should be spoken)

This change fixes row heading that is not directly related or labeled, like:
```
<h3 class="h2">Recently Uploaded</h3>
<div role="row" data-section="5">
...

or 
<div class="h2" role="heading" aria-level="2">Trending</div>
  <div role="row" data-section="6">
    <div role="gridcell" tabindex="0">
    <div class="vid-title">Excellent Throws Every Time</div>
    </div>

```

Comment (lauromoura) [reply Source/WebCore/accessibility/AccessibilityNodeObject.cpp:1459]: Thanks for the clarification. I think this PR can be merged as is and we can open a bug upstream to add a test/implement this.

**Pull request merged**